tlslite/tlsrecordlayer.py

```diff
@@ -833,6 +846,9 @@ def _getNextRecord(self):
             # application data isn't made out of messages, pass it through
             if header.type == ContentType.application_data:
                 yield (header, parser)
+            # heartbeat message isn't made out of messages, too
+            elif header.type == ContentType.heartbeat:
+                yield (header, parser)
```

Comment:
When we are reading messages from socket and we got heartbeat message, then we dont want let it to Defragmenter and just return header and parser. 
You wrote me in email, that I should handle it in `_getNextRecord()` like Application Data, so I inspired by previous condition. Without this when we receive heartbeat message, then it goes throught to Defragmenter and it cant handle it.